draft-nro-bulk-rdap.md

```diff
@@ -48,184 +48,288 @@ feature of this service.
 
 # Bulk Data Format {#bulk_data_format}
 
-The data returned for a Bulk RDAP request ((#bulk_rdap_url)) is a JSON object comprising JSON members for metadata and
-JSON data for a single RDAP object class.
+The data returned for a Bulk RDAP request ((#bulk_rdap_url)) is a JSON object providing metadata information, followed
+by newline-separated data objects for one or more RDAP object classes ([@!RFC9083, section 5]).
```

Comment:
Do you mean JSON Text Sequences [RFC7464] format ? I think this RFC serves our purpose.

Reply:
IIUC, our current bulk data structure aligns more with JSONL (JSON Lines) than JSON Text Sequences. Though I don't have any implementation experience with JSON Text Sequences, from what I gather, because of that additional non-printable  Record Separator (RS) character as a prefix for every JSON text (beside the line feed character at the end of it), viewing and storing the data becomes problematic. Also, unlike JSONL processing, a bulk data client would need to write a special parser to detect the RS character before each JSON text. (https://dbconvert.com/blog/json-lines-data-stream/amp/ seems to explain this JSONL versus JSON Text Sequences comparison well.)

That does raise one interesting question though. JSON Text Sequences comes with the "application/json-seq" media type. As for JSONL, there is none standardized at the IETF level but since we would only be streaming one JSON object per line feed, looks like application/json should suffice. Thoughts?

Reply:
We are already using JSON Text sequence in NRTMV4 so I thought it also makes sense to use here and with record separator (RS) making it more clear when there is new record but I agree with you regarding parsing (I doubt there would be many tools or libraries available to support this special character although I see one in npm https://www.npmjs.com/package/json-text-sequence). 

On the other hand, there are few attributes in DB that supports multi-line. Either we can escape them (if it is important to preserve) or join them all together to form one line.

Thanks for giving it a thought  though :)